kore/src/Kore/Log.hs

```diff
@@ -203,22 +208,24 @@ writeTChanLogger tChan =
 makeKoreLogger
     :: forall m
     .  MonadIO m
-    => TimestampsSwitch
+    => ExeName
+    -> TimestampsSwitch
     -> LogAction m Text
     -> LogAction m SomeEntry
-makeKoreLogger timestampSwitch logToText =
+makeKoreLogger exeName timestampSwitch logToText =
     Colog.cmapM withTimestamp
     $ contramap messageToText logToText
   where
     messageToText :: WithTimestamp -> Text
     messageToText (WithTimestamp entry localTime) =
         Pretty.renderStrict
         . Pretty.layoutPretty Pretty.defaultLayoutOptions
-        $ timestamp <> defaultLogPretty entry
+        $ timestamp <> exeName' <> Pretty.space <> defaultLogPretty entry
       where
         timestamp = case timestampSwitch of
             TimestampsEnable -> Pretty.brackets (formattedTime localTime)
             TimestampsDisable -> mempty
+        exeName' = Pretty.space <> Pretty.pretty exeName <> Pretty.colon
```

Comment:
Let's put the name before the timestamp and remove the space before it so that it is always aligned with the left-most column.